pkg/cloudprovider/providers/oci/load_balancer_spec.go

```diff
@@ -181,6 +185,10 @@ const (
 	// ServiceAnnotationNetworkLoadBalancerInitialFreeformTagsOverride is a service annotation for specifying
 	// freeform tags on the nlb
 	ServiceAnnotationNetworkLoadBalancerInitialFreeformTagsOverride = "oci-network-load-balancer.oraclecloud.com/freeform-tags"
+
+	// ServiceAnnotationNetworkLoadBalancerNodeFilter is a service annotation to select specific nodes as your backend in the NLB
+	// based on label selector.
+	ServiceAnnotationNetworkLoadBalancerNodeFilter = "oci-network-load-balancer.oraclecloud.com/node-label-selector"
```

Comment:
Please add the annotation to the list of NLB annotations here - https://github.com/oracle/oci-cloud-controller-manager/blob/master/docs/load-balancer-annotations.md
The description, default value and what values it can take.